lte/gateway/c/sctpd/src/sctp_connection.cpp

```diff
@@ -184,6 +184,13 @@ SctpStatus SctpConnection::HandleSendFailure(
   return SctpStatus::FAILURE;
 }
 
+SctpStatus SctpConnection::HandleDryEvent(
+    int sd, struct sctp_sender_dry_event* dry_event) {
+  MLOG(MDEBUG) << "Received SCTP_SENDER_DRY_EVENT for sd: " << sd
+               << " assoc_id:" << dry_event->sender_dry_assoc_id;
+  return SctpStatus::OK;
+}
+
```

Comment:
Idea is not to get the event if its not registered. If not registered it should go to default handler.

Reply:
Latest patch comment seems comments addressed.